src/cascadia/TerminalApp/TerminalPage.cpp

```diff
@@ -182,6 +182,30 @@ namespace winrt::TerminalApp::implementation
         _showDialogHandlers(*this, dialog);
     }
 
+    // Method Description:
+    // - Displays a dialog for warnings found while closing the terminal app using
+    // - key binding with multiple tabs opened. Display messages to warn user
```

Comment:
_uber-nit_: usually for doc comments, we'll only put the hyphen at the start of a sentence or bullet point, and just leave spaces at the start of following lines. For ex, I'd change this to:

```c++
// Method Description:
// - Displays a dialog for warnings found while closing the terminal app using
//   key binding with multiple tabs opened. Display messages to warn user that
//   more than 1 tabs are opend, and once the user click the OK button, remove
//   all the tabs and shut down and app. If cancel is clicked, the dialog will
//   close
// - Only one dialog can be visible at a time. If another dialog is visible
//   when this is called, nothing happens. See _ShowDialog for details

```

Reply:
(and similar for other doc comments added)

Reply:
removed